PerimeterXModule/PxModule.cs

```diff
@@ -366,6 +383,33 @@ public void Dispose()
 			}
 		}
 
+
+		/// <summary>
+		/// Checks the url if it should be a proxy request for the client/xhrs
+		/// </summary>
+		/// <param name="context">HTTP context for client</param>
+		private bool IsProxyRequest(HttpContext context)
```

Comment:
I would perhaps rename the method name to be a bit more descriptive in the context of first party, e.g. something like `IsFirstPartyProxyRequest`. Otherwise it may be misleading, as if there's some special generic proxy-related handling here.

Reply:
👍